pkgs/development/libraries/mesa/default.nix

```diff
@@ -136,6 +136,7 @@ self = stdenv.mkDerivation {
     meson pkg-config ninja
     intltool bison flex file
     python3Packages.python python3Packages.Mako
+    llvmPackages.llvm  # required for llvm-config-native
```

Comment:
It's sketch that this is in `nativeBuildInputs`, because we want `llvm-config-native` to be from `llvm` that runs on the host platform. I suspect this working because this might not be spliced for some reason, but that is not good. `llvmPackages.libllvm` up above should have `llvm-config-native` in its dev output.

Reply:
Oh! Yeah it's passed in top-level, should've thought about that. Not 100% sure how to correctly do this then?

Reply:
It's indeed probably because of something related to splicing (llvmPackages is overridden on top-level). I'll try if it helps to directly use llvmPackages_latest in the mesa package definition, we can then think up a solution based on that.